src/App.jsx

```diff
@@ -1,43 +1,17 @@
-import { useState, useEffect } from 'react';
+import { Route, Routes } from'react-router-dom';
 
-import { getFolder, getUser } from './apis/api';
-
-import Header from './Header';
-import Folder from './Folder';
-import Footer from './components/Footer';
-import Gnb from './components/Gnb';
+import SharedPage from './pages/SharedPage';
+import FolderPage from './pages/FolderPage';
 import './App.css';
 
 const App = () => {
-  const [users, setUsers] = useState({});
-  const [folders, setFolders] = useState({});
-
-  useEffect(() => {
-    const fetchFolder = async () => {
-      const { folder } = await getFolder();
-      
-      setFolders(folder);
-    };
-
-    fetchFolder();
-  }, []);
-
-  useEffect(() => {
-    const fetchUser = async () => {
-      const userInfo = await getUser();
-      
-      setUsers(userInfo);
-    };
-
-    fetchUser();
-  }, []);
 
   return (
     <div className='container'>
-      <Gnb items={users}/>
-      <Header items={folders}/>
-      <Folder items={folders}/>
-      <Footer/>
+      <Routes>
+        <Route path="shared" element={<SharedPage />} />
+        <Route path="folder" element={<FolderPage />} />
```

Comment:
사소하지만 슬래시를 붙이지 않는 건 보통 중첩 라우팅 경로를 작성할때 많이 써서 슬래시를 붙여서 절대 경로로 표현해주는게 조금 더 명확하게 느껴질 것 같기도 하네요~
```suggestion
        <Route path="/shared" element={<SharedPage />} />
        <Route path="/folder" element={<FolderPage />} />
```

Reply:
createBrowserRouter 함수를 이용해서 라우터를 구성해보셔도 좋을 것 같아요~
[참고](https://noah-dev.tistory.com/38#%F0%9F%9A%A8%20v6.4%EC%9D%98%20%EC%83%88%EB%A1%9C%EC%9A%B4%20router%EB%A1%9C%20%EC%97%85%EB%8D%B0%EC%9D%B4%ED%8A%B8%20%ED%95%98%EA%B8%B0-1)